clients/python/test/e2e/test_notebooks.py

```diff
@@ -60,6 +60,6 @@ def test_run_notebooks(tmp_path: Path, notebook: Path, params: dict[str, Any] =
         input_path=tmp_nb,
         output_path=output,
         kernel_name="python3",
-        parameters=params,
+        parameters=params or {},
```

Comment:
Wouldn't it make sense to then also make `params` input optional?

Reply:
AFAIK default input parameters in tests do not have any use. I just added that there in case you passed some sort of params that was null